huaweicloud/resource_huaweicloud_cce_node_pool.go

```diff
@@ -0,0 +1,600 @@
+						"size": {
+							Type:     schema.TypeInt,
+							Required: true,
+						},
+						"volumetype": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"extend_param": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+					}},
+			},
+			"data_volumes": {
+				Type:     schema.TypeList,
+				Required: true,
+				ForceNew: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"size": {
+							Type:     schema.TypeInt,
+							Required: true,
+						},
+						"volumetype": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"extend_param": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+					}},
+			},
+			"os": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"key_pair": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ForceNew:     true,
+				ExactlyOneOf: []string{"password", "key_pair"},
+			},
+			"password": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ForceNew:     true,
+				Sensitive:    true,
+				ExactlyOneOf: []string{"password", "key_pair"},
+			},
+			"taints": {
+				Type:     schema.TypeList,
+				Optional: true,
+				ForceNew: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"key": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"value": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"effect": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+					}},
+			},
+			"eip_ids": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				ForceNew: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				Set:      schema.HashString,
+				ConflictsWith: []string{
+					"eip_id", "iptype", "bandwidth_charge_mode", "bandwidth_size", "sharetype",
+				},
+				Deprecated: "use eip_id instead",
+			},
+			"eip_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				ConflictsWith: []string{
+					"eip_ids", "iptype", "bandwidth_charge_mode", "bandwidth_size", "sharetype",
+				},
+			},
+			"bandwidth_charge_mode": {
+				Type:          schema.TypeString,
+				Optional:      true,
+				ForceNew:      true,
+				ConflictsWith: []string{"eip_ids", "eip_id"},
+			},
+			"iptype": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				RequiredWith: []string{
+					"iptype", "bandwidth_size", "sharetype",
+				},
+			},
+			"sharetype": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				RequiredWith: []string{
+					"iptype", "bandwidth_size", "sharetype",
+				},
+			},
+			"bandwidth_size": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				ForceNew: true,
+				RequiredWith: []string{
+					"iptype", "bandwidth_size", "sharetype",
+				},
+			},
+			"billing_mode": {
```

Comment:
the billing_mod can only be 0 for nodepool, so we can mark it as a computed attribute.